includes/api/class-wc-admin-rest-onboarding-plugins-controller.php

```diff
@@ -206,13 +206,14 @@ public function connect_jetpack() {
 			return new WP_Error( 'woocommerce_rest_jetpack_not_active', __( 'Jetpack is not installed or active.', 'woocommerce-admin' ), 404 );
 		}
 
-		$redirect_url = esc_url_raw(
+		$next_step_slug = 'store-details';
```

Comment:
Side note: I know we talked a bit about extensibility before.  Would we ever want to allow hooking into change this step?